internal/vpn/client.go

```diff
@@ -58,21 +58,35 @@ func NewClient(cfg ClientConfig, l logrus.FieldLogger, conn net.Conn) (*Client,
 		return nil, fmt.Errorf("error getting STCP entities: %w", err)
 	}
 
-	stcprARip, err := stcprAddressResolverIPFromEnv()
-	if err != nil {
-		return nil, fmt.Errorf("error getting stcpr AR IP: %w", err)
+	var stcprARip net.IP
+	if _, ok := os.LookupEnv(STCPRAddressResolverAddrEnvKey); ok {
+		stcprARip, err = stcprAddressResolverIPFromEnv()
+		if err != nil {
+			return nil, fmt.Errorf("error getting stcpr AR IP: %w", err)
+		}
 	}
 
-	stcphARip, err := stcphAddressResolverIPFromEnv()
-	if err != nil {
-		return nil, fmt.Errorf("error getting stcph AR IP: %w", err)
+	var stcphARip net.IP
+	if _, ok := os.LookupEnv(STCPHAddressResolverAddrEnvKey); ok {
+		stcphARip, err = stcphAddressResolverIPFromEnv()
+		if err != nil {
+			return nil, fmt.Errorf("error getting stcph AR IP: %w", err)
+		}
 	}
 
-	directIPs := make([]net.IP, 0, 5+len(dmsgSrvAddrs)+len(stcpEntities))
-	directIPs = append(directIPs, dmsgDiscIP, tpDiscIP, rfIP, stcprARip, stcphARip)
+	directIPs := make([]net.IP, 0, 3+len(dmsgSrvAddrs)+len(stcpEntities))
```

Comment:
What do you think about creating a slice that contains `dmsgDiscIP, tpDiscIP, rfIP` and replacing `3` with its length? I'm afraid it may be easy to miss updating `3` if a service with a direct IP is added in the future.

Reply:
Done that